src/repository/DeviceKeyDatabaseRepository.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import type { IBaseDatabase, IDeviceKeyRepository } from '../types';
+
+class DeviceKeyDatabaseRepository implements IDeviceKeyRepository {
+  _database: IBaseDatabase;
+  _collectionName: string = 'deviceKeys';
+
+  constructor(database: IBaseDatabase) {
+    this._database = database;
+  }
+
+  deleteById = async (deviceID: string): Promise<void> =>
```

Comment:
there are a lot of places where I need to change `Id` to `ID`, I'll change it all at once sometime in the future.